addons/hearing/XEH_preInit.sqf

```diff
@@ -8,11 +8,14 @@ PREP_RECOMPILE_END;
 
 #include "initSettings.inc.sqf"
 
+[QGVAR(updateVolume), LINKFUNC(updateVolume)] call CBA_fnc_addEventHandler;
+
 ["CBA_loadoutSet", {
     params ["_unit", "_loadout", "_extendedInfo"];
     if (_extendedInfo getOrDefault ["ace_earplugs", false]) then {
         _unit setVariable ["ACE_hasEarPlugsIn", true, true];
-        [[true]] remoteExec [QFUNC(updateVolume), _unit];
+
+        [QGVAR(updateVolume), [true], _unit] call CBA_fnc_targetEvent;
```

Comment:
```suggestion
        [QGVAR(updateVolume), [[true]], _unit] call CBA_fnc_targetEvent;
```

Reply:
I tested both scenarios, it doesn't seem to make a difference. However, it does make more sense to update it.